tslint.json

```diff
@@ -0,0 +1,48 @@
+{
+  "extends": "tslint:recommended",
+  "rules": {
+    "indent": [ true, "tabs" ],
+    "jsdoc-format": true,
+    "whitespace": [
+      false,
+      "check-type"
+    ],
+    "object-literal-sort-keys": false,
+    "only-arrow-functions": [false],
+    "quotemark": [true, "single", "avoid-escape"],
+    "max-classes-per-file": [false],
+    "max-line-length": [180],
```

Comment:
This should be 120 AFAIK. My understanding is soft limit of 80 chars, hard limit of 120.

Reply:
:+1: changing that... Surprisingly prettier uses 120!